src/model/maps.mli

```diff
@@ -422,7 +422,7 @@ class virtual module_ : object
   method virtual signature : Signature.t -> Signature.t
 
   method virtual module_type_functor_arg :
-    FunctorArgument.t option -> FunctorArgument.t option
+    FunctorParameter.t -> FunctorParameter.t
 
```

Comment:
Should probably rename the method to use `param` instead of `arg` to match the type name change

Reply:
Done.